CHANGELOG

```diff
@@ -1,3 +1,9 @@
+* Sat May 19 2018 Trevor Vaughan <tvaughan@onyxpoint.com> - 7.1.0-0
+- Added support for setting dconf options for GDM
```

Comment:
Seems important to note that GDM settings previously in SIMP gnome module are now set here.

Reply:
90% sure that was in the writeup in the gnome module and, since that's where the issue was, I was thinking that should be sufficient.